javascript/javascript1/week4/hw4.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="hw4.js"></script>
+    <title>Document</title>
+</head>
+
+<body>
+
+</body>
+
+</html>
```

Comment:
Hello Kfazinic,
I relay get confused. Here I can see that I am in the javascript2/week2 and you have commented on this exercises, however the above comment is for Javascript1/week4.

Reply:
Hi @hamedveloper , yes - you submitted all these files in this homework PR so I reviewed them all even though I noticed some/most of them didn't correspond to that week's problems. I figured it can't hurt to get more comments. 😄
Be careful to commit only the relevant files in your future PRs (though I'm sure you're aware of that by now :)).

Reply:
I am trying to put them in the right place. anyway I am very thankfull for your comments.